source/_docs/eu-region.md

```diff
@@ -9,59 +9,119 @@ earlyaccess: true
+1.  Connect all custom domains to target environments (e.g., `www.example.com` and `example.com` on Live).
+2.  Configure DNS based on recommended values provided by the Site Dashboard's Domain tool.
 
-1.  Add domains to all environments.
-2.  Update DNS values to those provided by the Dashboard. Note that during early access, it may take up to one week for your domain to route directly to the EU region.
-3.  [Open a Support ticket](https://dashboard.pantheon.io/#support){.external} with the following template:
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">It can take up to one week for your domain to route directly to the EU region.</p>
+    </div>
 
-    ```nohighlight
-    We are preparing to go live for the site named <example>. Please add the following domains to your next EU deployment:
+3.  [Open a Support ticket](https://dashboard.pantheon.io/#support){.external} using the following template (replace `example` throughout):
 
-    example.com
-    www.example.com
-    blog.example.com
-    test.example.com
-    dev.example.com
+    >We are preparing to go live for the site named `<example>`. Please add the following domains to your next EU deployment:
+    >
+    >* example.com
+    >* www.example.com
+    >* blog.example.com
+    >* test.example.com
+    >* dev.example.com
+    > <br />
+    > <br />
+    >
+    >We understand that domains will be routed to the EU every Tuesday, for all requests submitted by the preceding Friday, and that we will receive confirmation when the update is complete, at which time we can update DNS.
+
+    <div class="alert alert-danger" role="alert">
+    <h4 class="warning">Warning</h4>
+    <p markdown="1">Do not drive traffic to custom domains (for example, via targeted ad campaigns or newsletter distribution lists) until Pantheon confirms the domain is configured for EU routing.</p>
+    </div>
 
-    We understand that domains will be routed to the EU every Tuesday, for all requests submitted by the preceding Friday, and that we will receive confirmation when the update is complete, at which time we can update DNS.
-    ```
 4. After receiving confirmation, celebrate. Your site is running in the EU!
 
+
+## Frequently Asked Questions
+### Can I use the EU region for an existing site?
+Yes, however you must migrate your existing site to a new Site Dashboard that was configured for the EU region during creation (as described [above](#create-a-new-site-required)).
+
+Contact your account owner or our [Sales team](https://pantheon.io/contact-us){.external} to learn about Pantheon's migration services or review [relaunch procedure](/docs/relaunch/) for steps on how to migrate the site yourself.
+
 ## Coming Soon
 
 More EU Region features are in active development. [Contact us](https://pantheon.io/contact-us){.external} to learn more, and check this guide for updates.
```

Comment:
- Disaster Recovery for EU Region
- Region Facet on Organization Dashboard
- More Regions!  Tell us about your needs for additional regions by filling out this survey: https://www.getfeedback.com/r/hkR9uTAJ

Also add separate section heading or FAQ:
- When will EU Region be available for sites paid by credit card?
General Availability is planned for late 2019. Tell us more about your needs by filling out this survey: https://www.getfeedback.com/r/hkR9uTAJ

add FAQ about
-  When will site creation through the dashboard be  available?
- General Availability is planned for late 2019.